classes/Data/ChangedItems.php

```diff
@@ -27,7 +33,7 @@ public function toArray(): array
             $changedItems[$url] = (array) $changedItem;
         }
 
-        return $changedItem;
```

Comment:
.. or maybe it was this one

Reply:
Or maybe both... 

Some say you shouldn't use plural forms of variables by adding an `s`. Too prone to mistakes...

Although strong typed languages would have prevented the mistake.